gum/backend-x86/gumstalker-x86.c

```diff
@@ -4625,91 +4646,114 @@ gum_exec_block_is_direct_jmp_to_plt_got (GumExecBlock * block,
                                          GumGeneratorContext * gc,
                                          GumBranchTarget * target)
 {
-  gboolean result = FALSE;
-
 #if defined (HAVE_LINUX)
   GumExecCtx * ctx = block->ctx;
   const cs_insn * insn = gc->instruction->ci;
-  GumModuleMap * map = NULL;
-  const GumModuleDetails * module;
-  GumElfModule * elf = NULL;
-  GumCheckElfSection plt_got = {
-    .name = ".plt.got",
-    .target = target,
-    .found = FALSE
-  };
-  GumCheckElfSection plt_sec = {
-    .name = ".plt.sec",
-    .target = target,
-    .found = FALSE
-  };
+  GArray * ranges;
+  GumMemoryRange *range;
```

Comment:
```suggestion
  GumMemoryRange * range;
```

But this should be moved to the inner scope where it is needed.